apps/ideaspace/src/components/modules/WorkshoppingPage/IdeaOverview/IdeaOverview.js

```diff
@@ -13,6 +13,10 @@ import {
 } from './StyledComponents';
 
 export const IdeaOverview = ({ selectedCard }) => {
+  console.log('selectedCard', selectedCard);
+
+  if (selectedCard.ideaName == "") return null;
```

Comment:
=== would be bettter to compare data type and value, unless there's a specific reason for using ==